README.md

```diff
@@ -1,6 +1,209 @@
+
+To run this test case, install TOSS and setup the environment as presented above, then run the following:
+```
+OMP_NUM_THREADS=1 python toss.py
+```
+TOSS will then return the four csv files
+- **run_time**: which refers to the total simulation time.
+- **champion_f**: being the fitness value corresponding to the champion chromosome.
+- **champion_x**: being the champion chromosome corresponding to the optimal set of trajectories.
+- **fitness_list**: which is a list of champion fitness values correponding to each generation.
+
+### Use Case 2: Trajectory Optimization Around an Arbitrary Clestial Body Maximising Gravitation Signal
+**[Coming Soon]**
+
+### Use Case 3: Trajectory Optimization with User-Defined Scientific Objective
+**[Coming Soon]**
+
+
+<!-- ROADMAP -->
+
+## Roadmap
+
+See the ###(https://github.com/rasmusmarak/TOSS/issues) for a list of proposed features (and known issues).
+
+
+<!-- CONTRIBUTING -->
+
+## Contributing
+
+The project is open to community contributions. Feel free to open an [issue](https://github.com/rasmusmarak/TOSS/issues) or write us an email if you would like to discuss a problem or idea first.
+
+If you want to contribute, please
+
+1. Fork the project on [GitHub](https://github.com/rasmusmarak/TOSS).
+2. Get the most up-to-date code by following this quick guide for installing TOSS from source:
+   1. Get [miniconda](https://docs.conda.io/en/latest/miniconda.html) or similar
+   2. Clone the repo
+   ```sh
+   git clone https://github.com/rasmusmarak/TOSS
+   ```
+   3. Set up the environment. This creates a conda environment called
+      `toss` and installs the required dependencies.
+   ```sh
+   conda env create -f environment.yml
+   conda activate toss
+   ```
+
+Once the installation is done, you are ready to contribute.
+Please note that Pull-Requests should be created from and into the `main` branch.
+
+3. Create your Feature Branch (`git checkout -b feature/yourFeature`)
+4. Commit your Changes (`git commit -m 'Add some yourFeature'`)
+5. Push to the Branch (`git push origin feature/yourFeature`)
+6. Open a Pull Request on the `main` branch.
+
+and we will have a look at your contribution as soon as we can.
+
+Furthermore, please make sure that your Pull-Request passes all automated tests. Review will only happen after that.
+Only Pull-Requests created on the `main` branch with all tests passing will be considered.
+
+<!-- LICENSE -->
+
+## License
+
+**[Coming Soon]** Distributed under the .... License. See [LICENSE]() for more information.
```

Comment:
need to update

Reply:
Referenced in issue #57 and added in PR #58.